skills/core.go

```diff
@@ -36,3 +40,37 @@ type Slot struct {
 	Attrs map[string]interface{}
 	Apps  []string
 }
+
+// SecuritySystem is a name of a security system.
+type SecuritySystem string
+
+// Type describes a group of interchangeable capabilities with common features.
+// Types are managed centrally and act as a contract between system builders,
+// application developers and end users.
+type Type interface {
+	// Unique and public name of this type.
+	Name() string
+	// Sanitize checks if a skill is correct, altering if necessary.
```

Comment:
(nitpick^2) I would use a \n before the comment here to separate the comment/func pairs from each other. But that may be just me.